test/main.lua

```diff
@@ -162,6 +245,7 @@ function build_test_env()
             mode = 'none',
         },
         expect = {},
+        expect_raw = expect,
```

Comment:
I would suggest importing `test/main.lua` from the test instead, since `expect` is a global already. It would need to be [tweaked slightly](https://docs.dfhack.org/en/stable/docs/Lua%20API.html#importing-scripts) to only call `main()` when invoked as a command, but I think it should work when that's done.

Reply:
good idea. I rewrote main.lua to be importable.